x/twap/api_test.go

```diff
@@ -173,7 +226,7 @@ func (s *TestSuite) TestGetArithmeticTwap() {
 			input:        makeSimpleTwapInput(baseTime.Add(-time.Hour), baseTime, quoteAssetA),
 			expErrorStr:  "too old",
 		},
-		// TODO: overflow tests, multi-asset pool handling, make more record interpolation cases
+		// TODO: overflow tests
```

Comment:
```suggestion
		// TODO: overflow tests, multi-asset pool handling
```

We still haven't tested things with multi-asset pools / records, and that we do what we expect! We can make a separate test instead of making this test larger for that though.